src/main/java/pl/cleankod/ApplicationInitializer.java

```diff
@@ -64,13 +87,53 @@ FindAccountAndConvertCurrencyUseCase findAccountAndConvertCurrencyUseCase(
     }
 
     @Bean
-    AccountController accountController(FindAccountAndConvertCurrencyUseCase findAccountAndConvertCurrencyUseCase,
-                                        FindAccountUseCase findAccountUseCase) {
-        return new AccountController(findAccountAndConvertCurrencyUseCase, findAccountUseCase);
+    AccountController accountController(FindAccountService findAccountService) {
+        return new AccountController(findAccountService);
     }
 
     @Bean
     ExceptionHandlerAdvice exceptionHandlerAdvice() {
         return new ExceptionHandlerAdvice();
     }
+
+    @Bean
+    public SimpleModule singleValueObjectModule() {
+        SimpleModule module = new SimpleModule();
+        module.addSerializer(SingleValueObject.class, new JsonSerializer<>() {
+            @Override
+            public void serialize(
+                    SingleValueObject singleValueObject,
+                    JsonGenerator jsonGenerator,
+                    SerializerProvider serializerProvider
+            ) throws IOException {
+                jsonGenerator.writeObject(singleValueObject.value());
+            }
+        });
+        module.addDeserializer(Account.class, new AccountDeserializeService());
```

Comment:
Does this mean that every new object, that we'd like to have deserialised, would have to be registered here separately? Isn't there a more future-proof and bigger-picture solution we could be aiming for?